2018-09-29 11:53:22.520 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 11:53:22.523 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 11:54:22.521 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 11:54:22.524 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 11:55:21.280 25420 DEBUG oslo_concurrency.lockutils [req-a31a2308-353b-4b24-bfe9-82368ad1946c 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:21.662 25420 DEBUG neutron.agent.dhcp.agent [req-17bb0d9f-032c-4059-a4d4-8efcd70a714b - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:55:21.664 25420 DEBUG neutron.agent.linux.utils [req-17bb0d9f-032c-4059-a4d4-8efcd70a714b - - - - -] Unable to access /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:21.665 25420 DEBUG neutron.agent.linux.dhcp [req-17bb0d9f-032c-4059-a4d4-8efcd70a714b - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d on network 07d8389b-6aac-428f-aaa1-27debd8de16d does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-29 11:55:21.665 25420 DEBUG neutron.agent.linux.dhcp [req-17bb0d9f-032c-4059-a4d4-8efcd70a714b - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d on network 07d8389b-6aac-428f-aaa1-27debd8de16d does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-29 11:55:22.521 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 11:55:22.524 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 11:55:24.087 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.109 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.123 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.141 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.152 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.166 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'tapd0003742-ce', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.186 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-o', 'link', 'show', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.199 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 11:55:24.220 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tapd0003742-ce) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 11:55:24.222 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-29 11:55:24.223 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tapd0003742-ce) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 11:55:24.224 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tapd0003742-ce) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 11:55:24.226 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'd0003742-cec2-4e5a-9615-3cdc126e568b', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:56:57:5c'})), record=tapd0003742-ce) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 11:55:24.254 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapd0003742-ce', 'address', 'fa:16:3e:56:57:5c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.266 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.276 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapd0003742-ce', 'netns', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.298 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'tapd0003742-ce', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.310 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'tapd0003742-ce', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.325 25420 DEBUG oslo_concurrency.lockutils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Acquired semaphore "iptables-qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:24.327 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.347 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.363 25420 DEBUG neutron.agent.linux.iptables_manager [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-29 11:55:24.364 25420 DEBUG oslo_concurrency.lockutils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Releasing semaphore "iptables-qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:24.365 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-o', 'link', 'show', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.377 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'addr', 'show', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.387 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'addr', 'add', '11.130.1.2/24', 'scope', 'global', 'dev', 'tapd0003742-ce', 'brd', '11.130.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.397 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.427 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Setting IPv4 gateway for dhcp netns on net 07d8389b-6aac-428f-aaa1-27debd8de16d to 11.130.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-29 11:55:24.429 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.130.1.1', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.440 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.449 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Cleaning stale devices for network 07d8389b-6aac-428f-aaa1-27debd8de16d _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-29 11:55:24.451 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.460 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.476 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Building initial lease file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-29 11:55:24.477 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Done building initial lease file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/leases with contents:
1538322924 fa:16:3e:56:57:5c 11.130.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-29 11:55:24.478 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:55:24.479 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:55:24.482 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Unable to access /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:24.483 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host', '--addn-hosts=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tapd0003742-ce', '--dhcp-range=set:tag0,11.130.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:24.513 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:24.513 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:24.514 25420 DEBUG neutron.agent.linux.external_process [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:55:24.515 25420 DEBUG oslo_concurrency.lockutils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:24.516 25420 DEBUG oslo_concurrency.lockutils [req-a31a2308-353b-4b24-bfe9-82368ad1946c 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:25.256 25420 DEBUG neutron.agent.dhcp.agent [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:55:25.258 25420 DEBUG neutron.agent.linux.dhcp [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:55:25.262 25420 DEBUG neutron.agent.linux.dhcp [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:55:25.272 25420 DEBUG neutron.agent.linux.utils [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:25.289 25420 DEBUG neutron.agent.linux.dhcp [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:55:25.290 25420 DEBUG neutron.agent.linux.utils [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:25.302 25420 DEBUG neutron.agent.linux.utils [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:25.315 25420 DEBUG neutron.agent.linux.utils [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:25.315 25420 DEBUG neutron.agent.linux.utils [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:25.316 25420 DEBUG neutron.agent.linux.external_process [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:55:25.317 25420 DEBUG oslo_concurrency.lockutils [req-04dcaf94-7ed1-4907-bb1d-af6a4167a035 - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:25.318 25420 DEBUG oslo_concurrency.lockutils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:25.318 25420 DEBUG neutron.agent.dhcp.agent [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T15:55:21Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.2'}], id=d0003742-cec2-4e5a-9615-3cdc126e568b, mac_address=fa:16:3e:56:57:5c, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:55:22Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-29 11:55:25.319 25420 DEBUG oslo_concurrency.lockutils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:25.320 25420 DEBUG oslo_concurrency.lockutils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:25.321 25420 INFO neutron.agent.dhcp.agent [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T15:55:21Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.2'}], id=d0003742-cec2-4e5a-9615-3cdc126e568b, mac_address=fa:16:3e:56:57:5c, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:55:23Z
2018-09-29 11:55:25.323 25420 DEBUG neutron.agent.dhcp.agent [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:55:25.324 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:55:25.327 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:55:25.333 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:25.345 25420 DEBUG neutron.agent.linux.dhcp [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:55:25.347 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:25.358 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:25.378 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:25.384 25420 DEBUG neutron.agent.linux.utils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:25.385 25420 DEBUG neutron.agent.linux.external_process [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:55:25.385 25420 DEBUG oslo_concurrency.lockutils [req-b0be05a5-42b7-45f0-91e7-4ed5fc8bd6da - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:53.238 25420 DEBUG oslo_concurrency.lockutils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:53.238 25420 INFO neutron.agent.dhcp.agent [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T15:55:52Z, description=, device_id=0cd2d676-9598-4dd6-ad75-f93bbf8f6be5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.1'}], id=88ab0208-e894-47a8-863f-1b64764374c3, mac_address=fa:16:3e:3c:17:4a, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:55:52Z
2018-09-29 11:55:53.240 25420 DEBUG neutron.agent.dhcp.agent [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:55:53.241 25420 DEBUG neutron.agent.linux.dhcp [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:55:53.244 25420 DEBUG neutron.agent.linux.dhcp [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:55:53.249 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:53.265 25420 DEBUG neutron.agent.linux.dhcp [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:55:53.266 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:53.278 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:53.291 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:53.292 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:53.293 25420 DEBUG neutron.agent.linux.external_process [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:55:53.293 25420 DEBUG oslo_concurrency.lockutils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:54.278 25420 DEBUG oslo_concurrency.lockutils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:54.279 25420 INFO neutron.agent.dhcp.agent [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T15:55:52Z, description=, device_id=0cd2d676-9598-4dd6-ad75-f93bbf8f6be5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.1'}], id=88ab0208-e894-47a8-863f-1b64764374c3, mac_address=fa:16:3e:3c:17:4a, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:55:52Z
2018-09-29 11:55:54.280 25420 DEBUG neutron.agent.dhcp.agent [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:55:54.281 25420 DEBUG neutron.agent.linux.dhcp [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:55:54.283 25420 DEBUG neutron.agent.linux.dhcp [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:55:54.287 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:54.298 25420 DEBUG neutron.agent.linux.dhcp [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:55:54.299 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:54.309 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:54.321 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:54.322 25420 DEBUG neutron.agent.linux.utils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:54.323 25420 DEBUG neutron.agent.linux.external_process [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:55:54.324 25420 DEBUG oslo_concurrency.lockutils [req-888f280d-1451-47a3-8778-42ded49ec6b0 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:56.448 25420 DEBUG oslo_concurrency.lockutils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:56.448 25420 INFO neutron.agent.dhcp.agent [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T15:55:52Z, description=, device_id=0cd2d676-9598-4dd6-ad75-f93bbf8f6be5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.1'}], id=88ab0208-e894-47a8-863f-1b64764374c3, mac_address=fa:16:3e:3c:17:4a, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:55:53Z
2018-09-29 11:55:56.449 25420 DEBUG neutron.agent.dhcp.agent [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:55:56.451 25420 DEBUG neutron.agent.linux.dhcp [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:55:56.452 25420 DEBUG neutron.agent.linux.dhcp [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:55:56.456 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:56.469 25420 DEBUG neutron.agent.linux.dhcp [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:55:56.470 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:56.483 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:56.495 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:56.496 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:56.496 25420 DEBUG neutron.agent.linux.external_process [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:55:56.497 25420 DEBUG oslo_concurrency.lockutils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:55:57.304 25420 DEBUG oslo_concurrency.lockutils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:55:57.306 25420 INFO neutron.agent.dhcp.agent [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T15:55:52Z, description=, device_id=0cd2d676-9598-4dd6-ad75-f93bbf8f6be5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.1'}], id=88ab0208-e894-47a8-863f-1b64764374c3, mac_address=fa:16:3e:3c:17:4a, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:55:57Z
2018-09-29 11:55:57.307 25420 DEBUG neutron.agent.dhcp.agent [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:55:57.309 25420 DEBUG neutron.agent.linux.dhcp [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:55:57.310 25420 DEBUG neutron.agent.linux.dhcp [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:55:57.313 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:57.327 25420 DEBUG neutron.agent.linux.dhcp [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:55:57.328 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:57.340 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:55:57.353 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:57.354 25420 DEBUG neutron.agent.linux.utils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:55:57.355 25420 DEBUG neutron.agent.linux.external_process [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:55:57.356 25420 DEBUG oslo_concurrency.lockutils [req-0548936d-3e39-4a07-b921-e8d7ac5359e2 - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:05.492 25420 DEBUG oslo_concurrency.lockutils [req-1ac90662-853a-4a90-a4d3-5b37745172ad 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Acquired semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:05.494 25420 DEBUG oslo_concurrency.lockutils [req-1ac90662-853a-4a90-a4d3-5b37745172ad 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Releasing semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:09.005 25420 DEBUG oslo_concurrency.lockutils [req-df850389-93cd-40b7-bee4-c1494baa4c7c - - - - -] Acquired semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:09.006 25420 DEBUG oslo_concurrency.lockutils [req-df850389-93cd-40b7-bee4-c1494baa4c7c - - - - -] Releasing semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:09.651 25420 DEBUG oslo_concurrency.lockutils [req-df850389-93cd-40b7-bee4-c1494baa4c7c - - - - -] Acquired semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:09.652 25420 DEBUG oslo_concurrency.lockutils [req-df850389-93cd-40b7-bee4-c1494baa4c7c - - - - -] Releasing semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:22.522 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 11:56:22.534 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.012s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 11:56:25.024 25420 DEBUG oslo_concurrency.lockutils [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:25.025 25420 INFO neutron.agent.dhcp.agent [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T15:56:23Z, description=, device_id=070c7a6e-ee9e-4ea0-bfd3-d112aeae9abe, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.9'}], id=1fe4cf08-c923-4dbd-92bd-57fc3bc9d65b, mac_address=fa:16:3e:85:8f:6d, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=True, project_id=16a78662a2764349adfd78804b4173a7, revision_number=3, security_groups=[u'087c9b17-57b8-4b66-a1d1-80fa37f21a01'], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:56:24Z
2018-09-29 11:56:25.026 25420 DEBUG neutron.agent.dhcp.agent [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:56:25.028 25420 DEBUG neutron.agent.linux.dhcp [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:56:25.033 25420 DEBUG neutron.agent.linux.dhcp [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:56:25.036 25420 DEBUG neutron.agent.linux.utils [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:25.056 25420 DEBUG neutron.agent.linux.dhcp [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:56:25.057 25420 DEBUG neutron.agent.linux.utils [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:25.070 25420 DEBUG neutron.agent.linux.utils [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:25.083 25420 DEBUG neutron.agent.linux.utils [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:56:25.084 25420 DEBUG neutron.agent.linux.utils [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:56:25.084 25420 DEBUG neutron.agent.linux.external_process [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:56:25.085 25420 DEBUG oslo_concurrency.lockutils [req-3773ee8c-5031-4d3f-bebb-66b1df357878 4b0d5a6b099e4bdf931f16be468d8423 16a78662a2764349adfd78804b4173a7 - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:27.486 25420 DEBUG oslo_concurrency.lockutils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:27.487 25420 INFO neutron.agent.dhcp.agent [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T15:56:23Z, description=, device_id=070c7a6e-ee9e-4ea0-bfd3-d112aeae9abe, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.9'}], id=1fe4cf08-c923-4dbd-92bd-57fc3bc9d65b, mac_address=fa:16:3e:85:8f:6d, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=True, project_id=16a78662a2764349adfd78804b4173a7, revision_number=4, security_groups=[u'087c9b17-57b8-4b66-a1d1-80fa37f21a01'], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:56:26Z
2018-09-29 11:56:27.488 25420 DEBUG neutron.agent.dhcp.agent [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:56:27.489 25420 DEBUG neutron.agent.linux.dhcp [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:56:27.495 25420 DEBUG neutron.agent.linux.dhcp [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:56:27.504 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:27.518 25420 DEBUG neutron.agent.linux.dhcp [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:56:27.519 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:27.531 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:27.547 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:56:27.549 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:56:27.550 25420 DEBUG neutron.agent.linux.external_process [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:56:27.550 25420 DEBUG oslo_concurrency.lockutils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:28.241 25420 DEBUG oslo_concurrency.lockutils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:28.241 25420 INFO neutron.agent.dhcp.agent [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T15:56:23Z, description=, device_id=070c7a6e-ee9e-4ea0-bfd3-d112aeae9abe, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.9'}], id=1fe4cf08-c923-4dbd-92bd-57fc3bc9d65b, mac_address=fa:16:3e:85:8f:6d, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=True, project_id=16a78662a2764349adfd78804b4173a7, revision_number=5, security_groups=[u'087c9b17-57b8-4b66-a1d1-80fa37f21a01'], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T15:56:27Z
2018-09-29 11:56:28.243 25420 DEBUG neutron.agent.dhcp.agent [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 11:56:28.244 25420 DEBUG neutron.agent.linux.dhcp [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 11:56:28.246 25420 DEBUG neutron.agent.linux.dhcp [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 11:56:28.250 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:28.264 25420 DEBUG neutron.agent.linux.dhcp [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 11:56:28.264 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:28.275 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 11:56:28.288 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:56:28.289 25420 DEBUG neutron.agent.linux.utils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 11:56:28.290 25420 DEBUG neutron.agent.linux.external_process [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 11:56:28.291 25420 DEBUG oslo_concurrency.lockutils [req-334ee51d-9087-45e7-87d5-33ee97f63eb1 d1138ec1c77349f9b95c6580aab47cb8 007b5c587a144e49a50c9008f27cd2bd - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:53.358 25420 DEBUG oslo_concurrency.lockutils [req-ec54b617-2c74-423b-8a13-3c4449d1272b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:53.359 25420 DEBUG oslo_concurrency.lockutils [req-ec54b617-2c74-423b-8a13-3c4449d1272b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Releasing semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:56:54.398 25420 DEBUG oslo_concurrency.lockutils [req-ec54b617-2c74-423b-8a13-3c4449d1272b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 11:56:54.399 25420 DEBUG oslo_concurrency.lockutils [req-ec54b617-2c74-423b-8a13-3c4449d1272b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Releasing semaphore "dhcp-agent-network-lock-9619842c-a15f-4600-bdce-bfe274d454b0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 11:57:22.522 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 11:57:22.524 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 11:58:22.522 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 11:58:22.525 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 11:59:22.522 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 11:59:22.525 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:00:17.984 25420 DEBUG oslo_concurrency.lockutils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:00:17.985 25420 DEBUG neutron.agent.dhcp.agent [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:00:17.986 25420 DEBUG neutron.agent.linux.utils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'dhcp_release', 'tapd0003742-ce', '11.130.1.9', 'fa:16:3e:85:8f:6d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:18.012 25420 DEBUG neutron.agent.linux.dhcp [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:00:18.016 25420 DEBUG neutron.agent.linux.dhcp [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:00:18.021 25420 DEBUG neutron.agent.linux.utils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:18.031 25420 DEBUG neutron.agent.linux.dhcp [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:00:18.033 25420 DEBUG neutron.agent.linux.utils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:18.045 25420 DEBUG neutron.agent.linux.utils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:18.061 25420 DEBUG neutron.agent.linux.utils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:00:18.062 25420 DEBUG neutron.agent.linux.utils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:00:18.063 25420 DEBUG neutron.agent.linux.external_process [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:00:18.063 25420 DEBUG oslo_concurrency.lockutils [req-1c41ba65-0e70-4171-9ab7-4bc1cef7553b aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:00:22.523 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:00:22.525 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:00:57.123 25420 DEBUG oslo_concurrency.lockutils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:00:57.124 25420 DEBUG neutron.agent.dhcp.agent [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:00:57.126 25420 DEBUG neutron.agent.linux.utils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'dhcp_release', 'tapd0003742-ce', '11.130.1.1', 'fa:16:3e:3c:17:4a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:57.143 25420 DEBUG neutron.agent.linux.dhcp [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:00:57.150 25420 DEBUG neutron.agent.linux.dhcp [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:00:57.158 25420 DEBUG neutron.agent.linux.utils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['kill', '-HUP', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:57.172 25420 DEBUG neutron.agent.linux.dhcp [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:00:57.174 25420 DEBUG neutron.agent.linux.utils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:57.188 25420 DEBUG neutron.agent.linux.utils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapd0003742-ce'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:00:57.201 25420 DEBUG neutron.agent.linux.utils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:00:57.201 25420 DEBUG neutron.agent.linux.utils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:00:57.202 25420 DEBUG neutron.agent.linux.external_process [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:00:57.202 25420 DEBUG oslo_concurrency.lockutils [req-87798fea-af9c-49c0-bcea-222b4d5e3b66 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:18.828 25420 DEBUG oslo_concurrency.lockutils [req-b52e90d5-facb-4e6a-be61-fda0803a8c82 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:01:18.829 25420 DEBUG neutron.agent.dhcp.agent [req-b52e90d5-facb-4e6a-be61-fda0803a8c82 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:01:18.832 25420 DEBUG neutron.agent.linux.utils [req-b52e90d5-facb-4e6a-be61-fda0803a8c82 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['kill', '-9', '960'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:18.848 25420 DEBUG neutron.agent.linux.utils [req-b52e90d5-facb-4e6a-be61-fda0803a8c82 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 12:01:18.876 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tapd0003742-ce) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:01:18.944 25420 DEBUG neutron.agent.linux.utils [req-656134e7-920c-45c0-8c63-b4efe00793de - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:18.955 25420 DEBUG neutron.agent.linux.utils [req-656134e7-920c-45c0-8c63-b4efe00793de - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:18.966 25420 DEBUG oslo_concurrency.lockutils [req-656134e7-920c-45c0-8c63-b4efe00793de - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:20.727 25420 DEBUG neutron.agent.dhcp.agent [req-9fcccdbb-d1ad-41f9-ad43-83572d77f502 - - - - -] resync (07d8389b-6aac-428f-aaa1-27debd8de16d): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-29 12:01:20.729 25420 INFO neutron.agent.dhcp.agent [req-9fcccdbb-d1ad-41f9-ad43-83572d77f502 - - - - -] Synchronizing state
2018-09-29 12:01:21.281 25420 INFO neutron.agent.dhcp.agent [req-1ac1cc6c-cc58-4cde-9dfa-b0c8dac8edfb - - - - -] All active networks have been fetched through RPC.
2018-09-29 12:01:21.283 25420 INFO neutron.agent.dhcp.agent [-] Starting network 07d8389b-6aac-428f-aaa1-27debd8de16d dhcp configuration
2018-09-29 12:01:21.283 25420 DEBUG neutron.agent.dhcp.agent [-] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:01:21.285 25420 DEBUG neutron.agent.linux.utils [-] Unable to access /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:21.286 25420 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d on network 07d8389b-6aac-428f-aaa1-27debd8de16d does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-29 12:01:21.286 25420 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d on network 07d8389b-6aac-428f-aaa1-27debd8de16d does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-29 12:01:21.291 25420 INFO neutron.agent.dhcp.agent [-] Starting network 9619842c-a15f-4600-bdce-bfe274d454b0 dhcp configuration
2018-09-29 12:01:21.293 25420 INFO neutron.agent.dhcp.agent [-] Finished network 9619842c-a15f-4600-bdce-bfe274d454b0 dhcp configuration
2018-09-29 12:01:22.524 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:01:22.525 25420 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:01:23.726 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.739 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.749 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.775 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.784 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.798 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'tapcad705d7-30', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.819 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-o', 'link', 'show', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.831 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 12:01:23.853 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tapcad705d7-30) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:01:23.854 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-29 12:01:23.855 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tapcad705d7-30) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:01:23.856 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tapcad705d7-30) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:01:23.857 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'cad705d7-30db-4011-9d4c-8fed5ca049ce', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:72:74:86'})), record=tapcad705d7-30) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:01:23.882 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapcad705d7-30', 'address', 'fa:16:3e:72:74:86'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.891 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.900 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapcad705d7-30', 'netns', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.917 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'tapcad705d7-30', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.935 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'link', 'set', 'tapcad705d7-30', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.948 25420 DEBUG oslo_concurrency.lockutils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Acquired semaphore "iptables-qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:01:23.949 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.965 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.983 25420 DEBUG neutron.agent.linux.iptables_manager [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-29 12:01:23.983 25420 DEBUG oslo_concurrency.lockutils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Releasing semaphore "iptables-qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:23.984 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-o', 'link', 'show', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:23.995 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', 'addr', 'show', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.006 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'addr', 'add', '11.130.1.2/24', 'scope', 'global', 'dev', 'tapcad705d7-30', 'brd', '11.130.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.016 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.026 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Setting IPv4 gateway for dhcp netns on net 07d8389b-6aac-428f-aaa1-27debd8de16d to 11.130.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-29 12:01:24.027 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.130.1.1', 'dev', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.038 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.048 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Cleaning stale devices for network 07d8389b-6aac-428f-aaa1-27debd8de16d _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-29 12:01:24.049 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.058 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.072 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Building initial lease file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-29 12:01:24.073 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Done building initial lease file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/leases with contents:
1538323284 fa:16:3e:72:74:86 11.130.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-29 12:01:24.074 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:01:24.074 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:01:24.076 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Unable to access /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:24.077 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host', '--addn-hosts=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tapcad705d7-30', '--dhcp-range=set:tag0,11.130.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.099 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:24.102 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:24.103 25420 DEBUG neutron.agent.linux.external_process [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:01:24.103 25420 INFO neutron.agent.dhcp.agent [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Finished network 07d8389b-6aac-428f-aaa1-27debd8de16d dhcp configuration
2018-09-29 12:01:24.104 25420 INFO neutron.agent.dhcp.agent [req-1ac1cc6c-cc58-4cde-9dfa-b0c8dac8edfb - - - - -] Synchronizing state complete
2018-09-29 12:01:24.105 25420 DEBUG oslo_concurrency.lockutils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:01:24.105 25420 INFO neutron.agent.dhcp.agent [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T16:01:21Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.2'}], id=cad705d7-30db-4011-9d4c-8fed5ca049ce, mac_address=fa:16:3e:72:74:86, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T16:01:22Z
2018-09-29 12:01:24.106 25420 DEBUG neutron.agent.dhcp.agent [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:01:24.107 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:01:24.110 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:01:24.115 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '4024'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.131 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:01:24.131 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.142 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.156 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:24.157 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:24.158 25420 DEBUG neutron.agent.linux.external_process [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:01:24.159 25420 DEBUG oslo_concurrency.lockutils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:24.160 25420 DEBUG oslo_concurrency.lockutils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:01:24.161 25420 INFO neutron.agent.dhcp.agent [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T16:01:21Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-07d8389b-6aac-428f-aaa1-27debd8de16d, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'e503eb83-a087-42ae-8d1a-a68535c53248', u'ip_address': u'11.130.1.2'}], id=cad705d7-30db-4011-9d4c-8fed5ca049ce, mac_address=fa:16:3e:72:74:86, name=, network_id=07d8389b-6aac-428f-aaa1-27debd8de16d, port_security_enabled=False, project_id=16a78662a2764349adfd78804b4173a7, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=16a78662a2764349adfd78804b4173a7, updated_at=2018-09-29T16:01:23Z
2018-09-29 12:01:24.162 25420 DEBUG neutron.agent.dhcp.agent [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:01:24.162 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Building host file: /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:01:24.164 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Done building host file /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:01:24.167 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '4024'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.178 25420 DEBUG neutron.agent.linux.dhcp [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Reloading allocations for network: 07d8389b-6aac-428f-aaa1-27debd8de16d reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:01:24.179 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-4', 'route', 'list', 'dev', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.191 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d', 'ip', '-6', 'route', 'list', 'dev', 'tapcad705d7-30'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:24.205 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:24.206 25420 DEBUG neutron.agent.linux.utils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:24.206 25420 DEBUG neutron.agent.linux.external_process [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:01:24.207 25420 DEBUG oslo_concurrency.lockutils [req-35a7dfe1-bce4-482b-a609-581918ca503a - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:27.158 25420 DEBUG oslo_concurrency.lockutils [req-08590997-b9e6-4da9-9574-2e512d34fef1 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:01:27.160 25420 DEBUG neutron.agent.dhcp.agent [req-08590997-b9e6-4da9-9574-2e512d34fef1 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:01:27.164 25420 DEBUG neutron.agent.linux.utils [req-08590997-b9e6-4da9-9574-2e512d34fef1 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command (rootwrap daemon): ['kill', '-9', '4024'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:27.183 25420 DEBUG neutron.agent.linux.utils [req-08590997-b9e6-4da9-9574-2e512d34fef1 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 12:01:27.203 25420 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tapcad705d7-30) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:01:27.262 25420 DEBUG neutron.agent.linux.utils [req-cd5fbf9f-70b0-4c4c-9008-f337f0ec4944 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:27.271 25420 DEBUG neutron.agent.linux.utils [req-cd5fbf9f-70b0-4c4c-9008-f337f0ec4944 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:27.279 25420 DEBUG oslo_concurrency.lockutils [req-cd5fbf9f-70b0-4c4c-9008-f337f0ec4944 - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:28.217 25420 DEBUG oslo_concurrency.lockutils [req-08590997-b9e6-4da9-9574-2e512d34fef1 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:01:29.874 25420 DEBUG neutron.agent.linux.utils [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:29.875 25420 DEBUG neutron.agent.linux.utils [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] Unable to access /var/lib/neutron/external/pids/07d8389b-6aac-428f-aaa1-27debd8de16d.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:29.875 25420 DEBUG neutron.agent.linux.external_process [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:01:29.876 25420 DEBUG neutron.agent.dhcp.agent [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] Calling driver for network: 07d8389b-6aac-428f-aaa1-27debd8de16d action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:01:29.877 25420 DEBUG neutron.agent.linux.utils [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] Unable to access /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:29.878 25420 DEBUG neutron.agent.linux.utils [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] Unable to access /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:01:29.878 25420 DEBUG neutron.agent.linux.external_process [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] No process started for 07d8389b-6aac-428f-aaa1-27debd8de16d disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:01:29.879 25420 DEBUG neutron.agent.linux.dhcp [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] Unable to access /var/lib/neutron/dhcp/07d8389b-6aac-428f-aaa1-27debd8de16d/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:271
2018-09-29 12:01:29.879 25420 DEBUG neutron.agent.linux.dhcp [req-7ca3329d-2204-4557-acb7-aa311a2562ae - - - - -] No interface exists for network 07d8389b-6aac-428f-aaa1-27debd8de16d destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1487
2018-09-29 12:01:29.907 25420 DEBUG neutron.agent.linux.utils [req-40279b03-620c-432c-8b9d-c57fee8b9063 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:01:29.923 25420 DEBUG neutron.agent.linux.dhcp [req-40279b03-620c-432c-8b9d-c57fee8b9063 - - - - -] Namespace already deleted: qdhcp-07d8389b-6aac-428f-aaa1-27debd8de16d _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:249
2018-09-29 12:01:29.924 25420 DEBUG oslo_concurrency.lockutils [req-40279b03-620c-432c-8b9d-c57fee8b9063 - - - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:29.925 25420 DEBUG oslo_concurrency.lockutils [req-08590997-b9e6-4da9-9574-2e512d34fef1 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Acquired semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:01:29.926 25420 DEBUG oslo_concurrency.lockutils [req-08590997-b9e6-4da9-9574-2e512d34fef1 aade3447accb4ba6835bc5ec3454f5a6 892b3d52c94348d8a7355f6390963dca - - -] Releasing semaphore "dhcp-agent-network-lock-07d8389b-6aac-428f-aaa1-27debd8de16d" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:01:54.105 25420 DEBUG neutron.agent.dhcp.agent [req-1ac1cc6c-cc58-4cde-9dfa-b0c8dac8edfb - - - - -] resync (07d8389b-6aac-428f-aaa1-27debd8de16d): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-29 12:01:54.106 25420 INFO neutron.agent.dhcp.agent [req-1ac1cc6c-cc58-4cde-9dfa-b0c8dac8edfb - - - - -] Synchronizing state
2018-09-29 12:01:54.758 25420 INFO neutron.agent.dhcp.agent [req-70309f52-27e7-402b-ba19-ebc8df113eda - - - - -] All active networks have been fetched through RPC.
2018-09-29 12:01:54.760 25420 INFO neutron.agent.dhcp.agent [-] Starting network 9619842c-a15f-4600-bdce-bfe274d454b0 dhcp configuration
2018-09-29 12:01:54.761 25420 INFO neutron.agent.dhcp.agent [-] Finished network 9619842c-a15f-4600-bdce-bfe274d454b0 dhcp configuration
2018-09-29 12:01:54.761 25420 INFO neutron.agent.dhcp.agent [req-70309f52-27e7-402b-ba19-ebc8df113eda - - - - -] Synchronizing state complete
